src/index.js

```diff
@@ -0,0 +1,292 @@
+          choices: generateEmployeeChoices(employees),
+        },
+      ];
+
+      const { employeeChoice, newManager } = await inquirer.prompt(
+        updateManager
+      );
+
+      await db.query(
+        `UPDATE company_db.employee SET manager_id = '${newManager}' WHERE (id = '${employeeChoice}');`
+      );
+    }
+    // if UPDATE EMPLOYEE ROLE, then give the user the choice to update the role of an employee
+    if (userAction === "Update Employee role") {
+      //prompt questions to user
+      const roles = await db.query("SELECT * FROM role");
+      const employees = await db.query("SELECT * FROM  employee");
+      const updateEmployeeInfo = [
+        {
+          type: "list",
+          message: "Select an employee to update their role",
+          name: "employeeToUpdate",
+          choices: generateEmployeeChoices(employees),
+        },
+        {
+          type: "list",
+          message: "Please select a role",
+          name: "employeeRole",
+          choices: generateRoleChoices(roles),
+        },
+      ];
+
+      const { employeeToUpdate, employeeRole } = await inquirer.prompt(
+        updateEmployeeInfo
+      );
+
+      await db.query(
+        `UPDATE company_db.employee SET role_id = '${employeeRole}' WHERE (id = '${employeeToUpdate}');`
+      );
+    }
+
+    // if View Employee by Manager, then give the user the choice to View Employee by Manager
+    if (userAction === "View Employee by Manager") {
+      //prompt questions to user
+      const viewEmployeeByManager = await db.query(employeeByManager);
+      console.table(viewEmployeeByManager);
+    }
+
+    // if Delete Record, then give the user the choice to Delete Record
+    if (userAction === "Delete Record") {
+      const roles = await db.query("SELECT * FROM role");
+      const employees = await db.query("SELECT * FROM  employee");
+      const department = await db.query("SELECT * FROM department");
+      //prompt questions to user
+      const { recordSelection } = await inquirer.prompt(deleteRecord);
+
+      if (recordSelection === "Delete employee") {
+        const employeeList = [
+          {
+            type: "list",
+            message: "Select an employee to delete:",
+            name: "employeeToDelete",
+            choices: generateEmployeeChoices(employees),
+          },
+        ];
+        const { employeeToDelete } = await inquirer.prompt(employeeList);
+
+        await db.query(
+          `DELETE FROM company_db.employee WHERE (id = '${employeeToDelete}');`
+        );
+        console.log(`Employee successfully deleted`);
+      }
+
+      if (recordSelection === "Delete role") {
+        const roleList = [
+          {
+            type: "list",
+            message: "Select a role to delete:",
+            name: "roleToDelete",
+            choices: generateRoleChoices(roles),
+          },
+        ];
+        const { roleToDelete } = await inquirer.prompt(roleList);
+
+        await db.query(
```

Comment:
You need to check here if a role has actually been deleted (somehow!). At the moment, the `db.query` isn't failing so it isn't reaching your `catch` block there, but departments still aren't being deleted yet we still get that confirmation console.log.

Reply:
Check what this query returns on a successful delete vs an unsuccesssful delete by storing the value in a variable and console log it. You may be able to use that to determine success or not.